index.jsx

```diff
@@ -69,15 +69,19 @@ class MapInteraction extends Component {
       btnClass: PropTypes.string,
       plusBtnClass: PropTypes.string,
       minusBtnClass: PropTypes.string,
-      controlsClass: PropTypes.string
+      controlsClass: PropTypes.string,
+      enableDrag: PropTypes.bool,
+      enableZoom: PropTypes.bool
```

Comment:
I think we should change these prop names to `disableDrag` and `disableZoom` and make them false by default. I think this more closely resembles how regular inputs work -- they are 'on' by default and you have to disable them explicitly like: `<input disabled={true} />`.

Also this is important because in the future we will have more fine grain options, such as `disableDragX`, `disableDragY`, etc.